src/components/app.tsx

```diff
@@ -328,12 +326,12 @@ export const App: React.FC<IAppProps<IModelInputState, IModelOutputState, IModel
           onChangeEnvironment={handleChangeEnvironment}
         />
       </div>
-      <ModalDialog
+      {/* <ModalDialog
         title={t("MACRO.ERROR.TITLE")}
         label={t("MACRO.ERROR.DESCRIPTION")}
         onClose={() => setShowModal(false)}
         showModal={showModal}
-      />
+      /> */}
```

Comment:
Should we just rip this out of the App component?

Reply:
The reason I left it in is that I still think we're going to need some sort of modal gating mechanism on rewind before we wipe out all the user's work. Uncommenting this code and turning it into an "Are you sure?" message might end up being the easiest path.

Reply:
Good call.  Let's leave it.  If we're going to warn there, I can see other places where we might warn as well.